game/src/Game/GameSession/NationManagementScreen/BudgetMenu.gd

```diff
@@ -41,8 +173,46 @@ func _on_update_active_nation_management_screen(active_screen : NationManagement
 	_update_info()
 
 func _update_info() -> void:
+	# TODO - remove _incVal and link the true data with the UI
+	_incVal += 1
+	
 	if _active:
-		# TODO - update UI state
+		_gold_label.set_text(str(_incVal - (_incVal % 7)) + "£")
+		_total_inc_label.set_text(str(_incVal) + "£")
+		_national_bank_label.set_text(str(_incVal * 2) + "£")
+		_total_funds_label.set_text(str(_incVal * 3) + "£")
+		_debt_val_label.set_text(str(_incVal * 4) + "£")
+		_interest_val_label.set_text(str(_incVal * 5) + "£")
+		_ind_sub_val_label.set_text(str(_incVal * 6) + "£")
+		_diplomatic_balance_label.set_text(str(_incVal * 8) + "£")
+		_total_exp_label.set_text(str(_incVal + 1) + "£")
+		_balance_label.set_text(str(_incVal) + "£")
```

Comment:
I personally prefer the functional versions, is there any meaningful difference between the two?

Reply:
The functions are overly verbose, properties are a standard of the language and we should get in the habit of using them, properties don't have consistent names and not all properties even have associated methods, especially if you make your own properties.

Also the formatting doesn't really matter if you use strings, however in this case `"%s£" % x` is better because it does not require `str`, values are automatically converted to strings via the formatting.